docs/sphinx/_toc.yml.in

```diff
@@ -125,3 +118,4 @@ subtrees:
         title: Providing feedback
```

Comment:
Provide to align with Contribute? There's no use of gerund for Contribute (not Contributing).

Reply:
Sure, makes sense to me. I'm okay with "Contribute"